build/Linux+BSD/mscore.desktop.in

```diff
@@ -10,6 +10,7 @@ Comment[fr]=Gravure de partitions musicales
 Exec=mscore@MSCORE_INSTALL_SUFFIX@ %F
 Icon=mscore@MSCORE_INSTALL_SUFFIX@
 StartupNotify=true
+StartupWMClass=mscore@MSCORE_INSTALL_SUFFIX@
```

Comment:
This change is in 3.5rc and in 3.x, so not an issue, but why then is it listed here?